corehq/pillows/mappings/domain_mapping.py

```diff
@@ -1,4 +1,4 @@
-DOMAIN_INDEX = "hqdomains_391fc13ad99b11e48da43c970e7be43b"
+DOMAIN_INDEX = "hqdomains_20150403_1448"
```

Comment:
@dimagi/team-commcare-hq this is a new convention i'm proposing to make indices more readable: [index]_[YYMMDD_HHMM]

there's no reason I can think of why they need to be guids.